kola-denylist.yaml

```diff
@@ -16,6 +16,8 @@
 
 - pattern: ext.config.rpm-ostree.replace-rt-kernel
   tracker: https://github.com/openshift/os/issues/1099
+  osversion:
+  - rhel-8.6
```

Comment:
IIUC we don't sync the denylist across branches here like we do for FCOS (though I think we should probably do that in the future). I think that means a rhel-8.6 would never get attempted to run against this code (since 4.14 is RHEL9 only) and you can just delete this.